src/components/HotspotActionSheet.tsx

```diff
@@ -234,8 +233,8 @@ const HotspotActionSheet = ({
 }
 
 const styles = StyleSheet.create({
-  cancelContainer: { backgroundColor: '#F0F0F5' },
-  cancelText: { color: '#B3B4D6' },
+  cancelContainer: { backgroundColor: '#02A8F5' },
```

Comment:
Should we be referencing styles from the theme? Will this work well with light v dark theme otherwise?

Reply:
I agree. I will make another push